hudi-cli/src/test/java/org/apache/hudi/cli/commands/TestFileSystemViewCommand.java

```diff
@@ -0,0 +1,245 @@
+import org.apache.hudi.common.table.view.SyncableFileSystemView;
+import org.apache.hudi.common.util.NumericUtils;
+
+import org.junit.Before;
+import org.junit.Test;
+import org.springframework.shell.core.CommandResult;
+
+import java.io.File;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.UUID;
+import java.util.function.Function;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertTrue;
+
+/**
+ * Test class for {@link FileSystemViewCommand}.
+ */
+public class TestFileSystemViewCommand extends AbstractShellIntegrationTest {
+  private String tablePath;
+  private String partitionPath;
+  private SyncableFileSystemView fsView;
+
+  private static String TEST_WRITE_TOKEN = "1-0-1";
+
+  @Before
+  public void init() throws IOException {
+    HoodieCLI.conf = jsc.hadoopConfiguration();
+
+    // Create table and connect
+    String tableName = "test_table";
+    tablePath = basePath + File.separator + tableName;
+    new TableCommand().createTable(
+        tablePath, tableName,
+        "COPY_ON_WRITE", "", 1, "org.apache.hudi.common.model.HoodieAvroPayload");
+
+    metaClient = HoodieCLI.getTableMetaClient();
+
+    partitionPath = HoodieTestCommitMetadataGenerator.DEFAULT_FIRST_PARTITION_PATH;
+    String fullPartitionPath = tablePath + "/" + partitionPath;
+    new File(fullPartitionPath).mkdirs();
+
+    // Generate 2 commits
+    String commitTime1 = "1";
+    String commitTime2 = "2";
+
+    String fileId1 = UUID.randomUUID().toString();
+
+    // Write date files and log file
+    new File(fullPartitionPath + "/" + FSUtils.makeDataFileName(commitTime1, TEST_WRITE_TOKEN, fileId1))
+        .createNewFile();
+    new File(fullPartitionPath + "/"
+        + FSUtils.makeLogFileName(fileId1, HoodieLogFile.DELTA_EXTENSION, commitTime1, 0, TEST_WRITE_TOKEN))
+        .createNewFile();
+    new File(fullPartitionPath + "/" + FSUtils.makeDataFileName(commitTime2, TEST_WRITE_TOKEN, fileId1))
+        .createNewFile();
+    new File(fullPartitionPath + "/"
+        + FSUtils.makeLogFileName(fileId1, HoodieLogFile.DELTA_EXTENSION, commitTime2, 0, TEST_WRITE_TOKEN))
+        .createNewFile();
+
+    // Write commit files
+    new File(tablePath + "/.hoodie/" + commitTime1 + ".commit").createNewFile();
+    new File(tablePath + "/.hoodie/" + commitTime2 + ".commit").createNewFile();
+
+    // Reload meta client and create fsView
+    metaClient = HoodieTableMetaClient.reload(metaClient);
+
+    fsView = new HoodieTableFileSystemView(metaClient, metaClient.getActiveTimeline(), true);
+  }
+
+  /**
+   * Test case for 'show fsview all'.
+   */
+  @Test
+  public void testShowCommits() {
+    // Test default show fsview all
+    CommandResult cr = getShell().executeCommand("show fsview all");
```

Comment:
ditto

Reply:
> ditto

@yanghua IMO, it is better to reuse title string like `Total Delta File Size`. Command string (`show fsview all`) maybe better to keep it the way it is, here and in `FileSystemViewCommand`, easier and clearer to known which command to run.

Reply:
OK, sounds reasonable.